app.py

```
@@ -61,8 +61,29 @@ def events():
+        try:
+            # Update your database insert to include phone
+            data = supabase.table('registrations').insert({
+                'name': name,
+                'email': email,
+                'phone': phone,
+                # ... other fields ...
+            }).execute()
+            
+            flash('Successfully joined the waitlist!', 'success')
+            return redirect(url_for('home'))
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)